### ISSUE TITLE
Bug: aria-extended giving invalid prop error

### ISSUE THREAD

**Issue by jugshaurya** [coding-blocks/gondor#72]
![extend](https://user-images.githubusercontent.com/39825643/82333189-0143f900-9a04-11ea-9c2c-2d8fdc658b2e.png)

**Comment by jugshaurya** [comment]
@hereisnaman, it should have been aria-expanded.
Will Submit a PR if you want me to solve this

**Comment by 0xnmn** [comment]
Go ahead

**Comment by jugshaurya** [comment]
@hereisnaman, Please Review
